python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -708,7 +708,12 @@ def _get_exposure_info(
                 extensionType.append("SCIENCE")
 
                 wcs = row.getWcs()
-                wcss.append(_get_wcs_from_sip(wcs))
```

Comment:
I think you can delete the `_get_wcs_from_sip` function completely now.

Reply:
I'm reluctant to remove this, as it was a pain to get it working properly and I'm not 100% certain it won't be needed in the future.